docs/documentation/pages/administrator/020-prerequisites.md

```diff
@@ -39,6 +39,16 @@ for avoiding confusing situations as outlined
 In the end, you won't need to create NGINX ingresses yourself: the Astarte Operator itself will take
 care of this task.
 
+
+## External RabbitMQ
+
+For production environments, consider using RabbitMQ deployed by the [RabbitMQ Cluster Operator]
+(https://www.rabbitmq.com/kubernetes/operator/operator-overview) or any other managed solution that 
+you prefer. This is due to the fact that RabbitMQ managed by the Astarte Operator is deprecated 
+starting from v24.5. Futher details can be found [here](https://github.com/astarte-platform/astarte-kubernetes-operator/issues/287). 
```

Comment:
```suggestion
you prefer. The Astarte Operator includes only basic management of RabbitMQ, which is deprecated since v24.5 and as such it should not be relied upon when dealing with production environments. Futher details can be found [here](https://github.com/astarte-platform/astarte-kubernetes-operator/issues/287). 
```